scripts/terraform-lint.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Just wondering if we need this? I just run `terraform fmt --recursive` and haven't needed anything different